internal/checker/cloudformation/rules/encryption/verify_es_encrypted_node.rego

```diff
@@ -0,0 +1,49 @@
+ # METADATA
+# title: "Verify that all information stored within Elasticsearch has node-to-node encryption enabled."
+# description: "This policy enables secure hosting of sensitive workloads by implementing encryption at rest. This feature safeguards data stored on instances within a cluster using encryption protocols, ensuring protection against unauthorized access." 
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticsearch-domain-nodetonodeencryptionoptions.html
+# custom:
+#   id: CB_CFT_004
+#   severity: MEDIUM
+package lib.cloudformation.CB_CFT_004
+
+import future.keywords.in
+
+resource := "AWS::Elasticsearch::Domain"
+
+is_valid {
+    some resources in input.Resources
+    resources.Type == resource
+}
+
```

Comment:
remove line 26 and just return the resources
pass[resources] {
    is_valid
    some resources in input.Resources
    node := resources.Properties.NodeToNodeEncryptionOptions
    node.Enabled == true
}